Scala/Scala.sublime-syntax

```diff
@@ -138,7 +138,7 @@ contexts:
           pop: true
         - match: '([a-zA-Z$_][a-zA-Z0-9$_]*)\s*:\s*'
           captures:
-            1: entity.name.variable variable.parameter
+            1: entity.name.parameter variable.parameter
```

Comment:
No, function parameters are never indexed to my knowledge. I've recently worked on a whole bunch, including C, C++, Go, Rust, Python, Ruby, Java, PHP, TCL and JavaScript. Parameters were never indexed in any of them before I started, or now.

It may just be that you make the decision to add a `.tmPreferences` file to you user package that turns on indexing/symbol list for the scope `source variable.parameter`.